src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          const result = this[i];
+
+          this[i] = this[j];
+          this[j] = result;
+        }
+      }
+    }
```

Comment:
The current implementation uses a bubble sort algorithm, which is not efficient for large arrays. Consider researching more efficient sorting algorithms like quicksort or mergesort for better performance.